test/client.test.ts

```diff
@@ -118,8 +118,8 @@ describe('Tests ReactNativeClient', () => {
       }).not.toThrow();
     });
 
-    test('falls back to YellowBox if no LogBox', async () => {
-      // @ts-ignore: Is Mocked
+    test.skip('falls back to YellowBox if no LogBox', async () => {
```

Comment:
Since we are planning to drop support for RN 0.63 we can remove this test. @marandaneto 

After the upgrade of RN the way how we clean `LogBox` doesn't work and I could not find a simple workaround.